bin/benchmark_pydy_code_gen.py

```diff
@@ -5,6 +5,8 @@
 
 # external libraries
 from numpy import hstack, ones, pi, linspace, array, zeros, zeros_like
+import matplotlib
+matplotlib.use('Agg')
```

Comment:
The benchmark doesn't call plt.show() and just saves the file so it's fine
as is. Works on my machine as is.

We can add the Travis check but I don't know the variable to check.
On Oct 17, 2015 21:31, "Jason K. Moore" <notifications@github.com> wrote:

> In bin/benchmark_pydy_code_gen.py
> <https://github.com/pydy/pydy/pull/277#discussion_r42313215>:
>
> > @@ -5,6 +5,8 @@
> >
> >  # external libraries
> >  from numpy import hstack, ones, pi, linspace, array, zeros, zeros_like
> > +import matplotlib
> > +matplotlib.use('Agg')
>
> Should this be "if on Travis use agg"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydy/pydy/pull/277/files#r42313215>.
>